e2e/package.json

```diff
@@ -18,6 +18,7 @@
         "@types/node": "^18.11.9"
     },
     "dependencies": {
+        "cuid": "^3.0.0",
```

Comment:
Not really an issue but I thought it might worth mentioning that JavaScript has it's own built in Crypto interface that can generate v4 uuids with 'randomUUID' method (since node v14).